packages/cli/src/utils/cli.ts

```diff
@@ -31,61 +30,47 @@ export async function displaySendTx<A>(
     const txReceipt = await txResult.waitReceipt()
     cli.action.stop()
 
-    if (displayEventName && txReceipt.events) {
-      Object.entries(txReceipt.events)
-        .filter(([eventName]) => eventName === displayEventName)
-        .forEach(([eventName, log]) => {
-          const { params } = parseDecodedParams((log as EventLog).returnValues)
-          console.log(chalk.magenta.bold(`${eventName}:`))
-          printValueMap(params, chalk.magenta)
-        })
+    if (txReceipt.events) {
+      let events: { [key: string]: any } = {}
+      Object.values(txReceipt.events).forEach(
+        (log) => (events[log.event] = parseDecodedParams(log.returnValues).params)
+      )
+      printValueMapRecursive(events, chalk.magenta)
     }
   } catch (e) {
     cli.action.stop(`failed: ${e.message}`)
     throw e
   }
 }
 
-export function printValueMap(valueMap: Record<string, any>, color = chalk.yellowBright.bold) {
-  console.log(
-    Object.keys(valueMap)
-      .map((key) => color(`${key}: `) + valueMap[key])
-      .join('\n')
-  )
-}
+export const printValueMap = (valueMap: Record<string, any>, color = chalk.yellowBright.bold) =>
```

Comment:
is there a reason to have this func in addition to `printValueMapRecursive`?

Reply:
I guess not
I think we can scrub the `Recursive` as its an unimportant detail of the implementation